quick-start-guides/how-to-connect-to-the-app.md

```diff
@@ -1,13 +1,101 @@
-# How to connect to the app
+# How to connect to the app?
 
-{% hint style="info" %}
-**Good to know:** your product docs aren't just a reference of all your features! use them to encourage folks to perform certain actions and discover the value in your product.
-{% endhint %}
+## Connecting to Buffer Finance - Arbitrum
 
-## The basics
+### Step 1: Setup Wallet
 
-Projects are containers for task lists. Think of them as a library for everything your team needs to get done to complete or ship a project.
+If you're new to DeFi, you will need to install and setup a non-custodial wallet, that allows you to directly connect and interact with the dApp. 
+For the purpose of this guide, we will be using MetaMask as an example.
+
+Follow this [guide](https://support.metamask.io/hc/en-us/articles/360015489531-Getting-started-with-MetaMask) to install and setup your MetaMask wallet.
+
+### Add Arbitrum Network to your wallet
```

Comment:
Network addition is automated by the code, user does not have to do it @RizzJ

Reply:
oh yes, thank you for pointing that out

corrected now